alchemist/alchemist-cognitive-agents/src/test/resources/feels-transmission-with-layer.yml

```diff
@@ -5,7 +5,7 @@ variables:
     formula: "\"danger\""
 
 environment:
-  type: Continuous2DEnvironment
+  type: EuclideanPhysics2DEnvironmentImpl
```

Comment:
Maybe Continuous2DEnvironment can become an interface extending PhysicsEnvironment like EuclideanPhysics2DEnvironment is now and EuclideanPhysics2DEnvironment can substitute EuclideanPhysics2DEnvironmentImpl. Obviously then more specific environments must extend EuclideanPhysics2DEnvironment rather than Continuous2DEnvironment.

Reply:
Another option is to simply include EuclideanPhysics2DEnvironmentImpl code inside Continuous2DEnvironment and delete EuclideanPhysics2DEnvironmentImpl. Nothing else would be necessary.

Reply:
Both make sense. Think about it and pick the one you deem best.
Also, change the default to point to that environment, if needed.